create-env.js

```diff
@@ -4,3 +4,6 @@ fs.copyFile('.env.example', '.env', (err) => {
   if (err) throw err;
   console.log('.env was created successfully');
 });
+const Handlebars = require("handlebars");
```

Comment:
Por que agregaste Handlebars aca? Veo que solo lo utilizas para el console log de abajo. El proyecto tiene un setup ya para poder mostrar todos los .hbs
Si quitas esto (y la dependencia del package.json) deberia funcionarte, no?

Reply:
agree, remove it